include/camp/resource.hpp

```diff
@@ -94,12 +94,36 @@ namespace resources
       void wait_for(Event *e) { m_value->wait_for(e); }
       void wait() { m_value->wait(); }
 
+      /*
+       * \brief Compares two Resources to see if they are equal.
+       *
+       * \return True or false depending on comparison with m_value if they are on the same platform.
```

Comment:
Better to describe the semantics of the comparison (i.e. what does equality mean) vs. the impl details?